### PR TITLE
feat: add translations (florian)

### LINKED ISSUE #29: feat: add dev locales (florian)
See https://github.com/CIDgravity/CIDgravity/issues/287#issuecomment-1259413154

@FlorianRuen 

- [x] Dashboard (Florian)
- [x] Clients (Florian)
- [x] Blacklist (Florian)
- [x] Playground (Florian)
- [x] Menu sidebar / topbar (Florian)
- [x] Help center (Florian)
- [ ] Wizard (Florian)

### PR DESCRIPTION
**Pull request by FlorianRuen**
Closes #29

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,8 @@
 cidgravity/node_modules
 cidgravity/update-locales.sh
 shared/node_modules
+<<<<<<< HEAD
```
Comment (nicobao) [inline .gitignore:4]: Merge conflict!

**Review by nicobao — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (FlorianRuen) [reply .gitignore]: Fixed !

**Review by nicobao — APPROVED** [review]
Good job!

**Pull request merged**